### ISSUE TITLE
Accessibility issue with screen reader and vscode keep saying name of project and WSL env (program title bar value)

### ISSUE THREAD

**Issue by dstpierre** [microsoft/vscode#221470]
Does this issue occur when all extensions are disabled?: Yes/No
yes

- VS Code Version: 1.91.0
- OS Version: WSL 2 / Ubuntu 20.04 / Windows 10

Steps to Reproduce:

1. Search by Name CTRL+P

or

1. Typing a function and open paren so the intellisence / function argument appear `abc(` 
Both triggers the same very verbose info which is not required as we're already in the app.

The screen reader will say:

[File name] - [Project name] - WLS Ubuntu 20.04 - Visual Srudio Code - Document

**And than will repeat that a second time** with "Application" instead of "Document".

And finally "Search file by name".

It's the same for the function arguments, it's saying all of this twice before finally starting to announce the arguments.

Basically the screen reader says the same as when the vscode application gets the main OS focus with the following difference:

When I ALT+TAB to vscode or return from a different virtual desktop it will say all info, but instead of "Document" will say "Desktop 2" and repeat with the "Application"

When ALT-Tabing I'm all good / can live with what the screen reader is saying.

But for CTRL+P and also for intellisence, is there a way to remove this long and superflus info?

Because I either have to wait to get the function arguments or the confirmation that CTRL+P is open, and if I hit a key to skip, I than need to redo the operation.

Thank you.

**Comment by dstpierre** [comment]
JFYI, same behavior with the latest 1.91.1 version.

**Comment by meganrogge** [comment]
On macOS (voiceover) / windows (nvda), I am not able to reproduce the issues you are describing, thus closing for now. @rperez030 , can you?

**Comment by rperez030** [comment]
I cannot reproduce it either, but I don't have WSL configured in my system at the moment. I tried with JAWS on Windows as well. @dstpierre which screen reader are you using?

**Comment by dstpierre** [comment]
Hey @meganrogge @rperez030 I [recorded an mp4 showcasing the issue](https://dominicstpierre.com/static/nvda_vscode_bug_showcase.mp4)

I'm using NVDA on Windows, maybe with that video this will help you reproduce the issue.

Hopefully it will as it is exrtremely disturbing.

Thanks again for the effort improving the a11y.

**Comment by meganrogge** [comment]
It looks like this is happening for you when you clear the line.

**Comment by dstpierre** [comment]
@meganrogge my naive impression is that it has to do with the main window thinking it's loosing focus and getting it back.

Because this happens when I do the following:

1. `CTRL=p` / open the fuzy file finder.
2. When the intellisense popup appears
3. When the hint popup appears for function arguments.

And since we know what it says, it keeps saying the main window titlebar text, pointing me in the direction where the window titlebar text might be annonced on a `focus` event and for some reasons doing the above action might be triggering a blur and than a focus causing the annoncement of the main window titlebar text.

It's annoncing exactly the same as when I `ALT+tab` and go back to vs code, but without ever leaving the application.

Is this window titlebar text a function, can it be traced, can it be made aware that the main application did not activate.

**Comment by meganrogge** [comment]
@jooyoungseo or @rperez030 have you experienced this? It does seem unexpected that something that loses focus would be announced by the screen reader

**Comment by dstpierre** [comment]
@meganrogge I was investigating this a bit, it seems directly related to this closed issue  #194255 if there were some settings to set the program titlebar this would at least remove this and would leave the current filename of the open file, which could be an liveable compromised. i.e. if I would be able to remove the text value of everything the screen reader says about the program titlebar and current file name each time there's suggestions poping or the CTRL+p dialog is displayed.

Any idea if vscode would allow me to do this, prevent any text value from being announce when it gets focus, because it's triggering the ~blur/focus (call it what you want) each time there something happening, and if there were a way to verride the program titlebar and the filename to empty string at least that would lower the noise.

Thoughts?
